workflow/common/util.go

```diff
@@ -263,6 +263,18 @@ func ProcessArgs(tmpl *wfv1.Template, args wfv1.ArgumentsProvider, globalParams,
 		if argParam != nil && argParam.Value != nil {
 			inParam.Value = argParam.Value
 		}
+
+		if inParam.ValueFrom != nil && inParam.ValueFrom.ConfigMapKeyRef != nil {
+			inParam.Value = wfv1.AnyStringPtr("Generated from configMap")
+			if clientset != nil {
+				configMapValue, err := util.GetConfigMaps(*clientset, namespace, inParam.ValueFrom.ConfigMapKeyRef.Name, inParam.ValueFrom.ConfigMapKeyRef.Key)
+				if err != nil {
+					return nil, errors.Errorf(errors.CodeBadRequest, "inputs.parameters.%s malformed. %s", inParam.Name, err)
```

Comment:
is this error message correct - could be other problems?

Reply:
Changed this to output the underlying error as well